pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -171,6 +187,7 @@ type SourceObjectKeySelector struct {
 
 	// Key of the entry in the object's `data` field to be used.
 	//+optional
+	// +kubebuilder:validation:MinLength=1
```

Comment:
@erikgb what does the combination `optional` and `MinLength=1` mean?
Do you have examples of yaml that works and does not work with this configuration + how it is different from eg. just `optional`?

Reply:
Just `optional` will allow the empty string. Which doesn't make sense here.